book/Makefile

```
@@ -15,20 +15,24 @@ upgrade:
 
 # Start a webserver serving the book locally
 serve:
-	cd website && jupyter-book start
+	cd en && jupyter-book start
 
 # Build the book and export as HTML
 book:
-	cd website && jupyter-book build --html
+	cd en && jupyter-book build --html
+	cd pt && jupyter-book build --html
 
 # Build the book stopping on errors
 strict:
-	cd website && jupyter-book build --html --strict
+	cd en && jupyter-book build --html --strict
+	cd pt && jupyter-book build --html --strict
 
 # Build the book in a non-interactive environment (for CI)
 ci:
-	cd website && jupyter-book build --html --ci
+	cd en && jupyter-book build --html --ci
+	cd pt && jupyter-book build --html --ci
 
 # Clean any build outputs and artifacts
 clean:
-	jupyter-book clean --all --yes ./website/_build
+	jupyter-book clean --all --yes ./en/_build
+	jupyter-book clean --all --yes ./pt/_build
```

```suggestion
	jupyter-book clean --all --yes ./**/_build
```
Would this not work?

---

Yes... sort of.

For reasons unclear to me I had to run it twice:
```
(venv) danny@hades:~/Documents/the-turing-way/book$ make clean
jupyter-book clean --all --yes ./**/_build
⛔️ pt/_build Error reading file: Error: EISDIR: illegal operation on a directory, read
⛔️ website/_build Error reading file: Error: EISDIR: illegal operation on a directory, read
Deleting all the following paths:

  - pt/_build/cache
  - pt/_build/html
  - pt/_build/templates
  - website/_build/cache
  - website/_build/html
  - website/_build/site
  - website/_build/templates

🗑  Deleting: pt/_build/cache
🗑  Deleting: pt/_build/html
🗑  Deleting: pt/_build/templates
🗑  Deleting: website/_build/cache
🗑  Deleting: website/_build/html
🗑  Deleting: website/_build/site
🗑  Deleting: website/_build/templates
(venv) danny@hades:~/Documents/the-turing-way/book$ make clean
jupyter-book clean --all --yes ./**/_build
⛔️ pt/_build Error reading file: Error: EISDIR: illegal operation on a directory, read
Deleting all the following paths:

  - pt/_build/site

🗑  Deleting: pt/_build/site
```

but I've noticed some weird behavior with `make clean` before (generally that EISDIR thing) so I suspect that's indpendent of this way of doing it.

---

Used here: https://github.com/the-turing-way/the-turing-way/pull/4290/commits/9831f0cfab92fe74f4ff4811fc56673486ebdac0